k8s/prometheus-operator/README.md

```diff
@@ -45,7 +49,7 @@ Configure `gcloud` as a Docker credential helper:
 gcloud auth configure-docker
 ```
 
-#### Create a Google Kubernetes Engine (GKE) cluster
+#### Create a Google Kubernetes Engine (GKE) cluster (you can skip this step if using an on-prem cluster)
```

Comment:
Let's put this in the text and change the wording. 

"Or you can use your existing [GKE on-prem](https://cloud.google.com/anthos/gke/docs/on-prem) cluster"

Reply:
done